### DISCUSSION TITLE
I need help with two customization CSS codes

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [Aris-t2/CustomCSSforFx#640] [Q&A]
## Description:

- I posted them a few weeks ago on Firefox UI Fix discussions, but I didn't get a response.
- I'm not friend with CSS, because I don't know much about programming. So, please write me the correct CSS code in a comment.

### My problem:
- I want to change the appearance of checkboxes to toggles without affecting the other checkboxes (like rounded checkboxes) using userChrome.css in Firefox.
- Also, does the CSS automatically tweak the space gap between the toggle and its title? Or do I need also to add a spacing code between the toggle and title?

#### **I mean something like this:**
- **Correct:** [toggle] Open previous windows and tabs
- **Incorrect:** [toggle] ----- Open previous windows and tabs
- The "-----" is the space gap, the "[toggle]" is the actual button, and the "Open previous windows and tabs" is the title

I wrote this originally on Mozilla support, but they didn't help me, and only told me, that I should go to Reddit, and I don't have a Reddit account.

### Screenshots:
![Help with changing appearance checkboxes](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/5c28db04-58e1-4e4f-b060-34e8d1011a25)

----------------

### My problem:
- I want to ask if you know how can I set the auto-hide "x" close tab button when only one tab is opened in Firefox.
- Also, how to set it to not count the Firefox View tab (button) as the second opened tab?

#### This is not a problem on Vivaldi, because there is already hidden the close tab button:
![image](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/fd566a80-1423-4d8c-9b54-5fe6c45f1a66)

#### Nor in Opera browser:
![image](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/86f9c0d5-caf6-409f-978c-10009454073c)

**Comment by Aleksandra0056** [comment]
I'm apologizing, that I created a new discussion, and didn't post my questions on the general discussion thread. I promise, that it never happens again.

**Comment by Aris-t2** [comment]
This area is mainly about discussing stuff/settings/options this project provides. There is a general thread for it here: https://github.com/Aris-t2/CustomCSSforFx/discussions/606

Changing the checkboxes to toggle is not trivial. Its not just an image swap. You have to inspect the code Firefox uses for both an try to port it. It might be time consuming and involve a lot of trial/error testing. This is most likely the main reason you don't get answers for this. Creating an Reddit account is as easy as creating a Github account, isn't it?

You can hide tab close button for one visible tab using this projects corresponding option: https://github.com/Aris-t2/CustomCSSforFx/blob/master/current/css/tabs/tab_close_hidden_for_only_one_visible_tab.css
